package.json

```diff
@@ -20,13 +20,14 @@
   "types": "types/index.d.ts",
   "scripts": {
     "dev": "concurrently \"npm run compile:deps -- --watch --env mode=development\" \"npm run compile:main -- --watch --env mode=development\"",
-    "compile": "npm run compile:deps -- --progress && npm run compile:main -- --progress",
+    "compile": "rimraf dist && npm run compile:deps -- --progress && npm run compile:main -- --progress",
     "compile:deps": "webpack --env buildDeps=true",
     "compile:main": "webpack",
     "test": "jest --config=jest.config.ts",
     "test:demo": "npm run test -- --demo=true --runInBand",
     "lint": "eslint --fix \"src/**/*.ts\" \"test/**/*.ts\" && prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "lint:check": "eslint \"src/**/*.ts\" \"test/**/*.ts\" && prettier --check \"src/**/*.ts\" \"test/**/*.ts\"",
+    "zip": "cd dist/ && bestzip ../extension.zip *",
```

Comment:
Is the zip needed for something related to development/production ? If not and this is just convenience function for sharing it with others, I would argue we should not pollute the repo.